test/unit/math/prim/prob/lkj_cov_test.cpp

```diff
@@ -85,7 +85,7 @@ TEST(ProbDistributionsLkjCov, ErrorChecks) {
 
   EXPECT_THROW(stan::math::lkj_cov_lpdf(Sigma, mu, sd, -1), std::domain_error);
   EXPECT_THROW(
-      stan::math::lkj_cov_lpdf(Sigma, muV, sd, stan::math::NOT_A_NUMBER),
+      stan::math::lkj_cov_lpdf(Sigma, mu, sd, stan::math::NOT_A_NUMBER),
```

Comment:
It is. However I reorder checks so that checks on sizes of the arguments are done before checks on values (which I think makes more sense). So this started throwing `std::invalid_argument`.

Anyway it is a bad test design to introduce two error conditions in the same test and expect exception for exately one of those.

Reply:
Got it. Makes sense.